docker/Dockerfile

```
@@ -1,3 +1,3 @@
-FROM python:3.10.6-slim-bullseye
+FROM python:3.10-slim-bullseye
```

```suggestion
FROM python:3.10.8-slim-bullseye
```

so we can quickly check which python version the image uses.

---

If you specify the python:3.10.8-slim-bullseye tag, then version 3.10.8 will be used, but if you specify the python:3.10-slim-bullseye tag, then the latest current version of python will always be used, at the moment it is 3.10.9.
Thus, it will not be necessary to change the minor version of the image every time - this process will occur automatically when a new version is released.

---

If we use `FROM python:3.10-slim-bullseye`, the python version installed in the image depends on when we build the image, right?

---

Maybe I'm too strict, but I want to eliminate ambiguity.

---

Right, the version depends on when we build the image.
python:3.10-slim-bullseye was created just in order not to change the minor version of python each time by hand, but to use the latest image available at the time of assembly.
If the person who collects the image needs a specific minor version, he will prescribe it, but such cases are much rarer than when a person needs the last available one.